### DISCUSSION TITLE
ESP32_S3 and C2 with the ST7789 display

### DISCUSSION THREAD

**Discussion by mike-rankin** [Bodmer/TFT_eSPI#1835] [Q&A - General] [answered]
I see Setup70b_ESP32_S3 and Setup70c_ESP32_C3 will now work for the ILI9341 displays but will the S3 and C3 work for the ST7789 displays if I modify the User_Setup.h and User_Setup_Select.h files? Right now I have the ESP32_S3 and ESP32_C3 hardware but only lcd displays with the ST7789 driver on them.

**Comment by Bodmer** [comment, marked as answer]
The provided setups are just examples, the C3 and S3 will work with most displays including ST7789.

**Reply by xavierzhu** [reply]
I tested the C3 with St7735 and ST7789.
ST7735 work fine,but ST7789 not work.

**Reply by Bodmer** [reply]
Try adding:
define TFT_SPI_MODE SPI_MODE0

To your setup.
